Makefile

```diff
@@ -266,6 +266,14 @@ install-schema: bins
 	./temporal-cassandra-tool --ep 127.0.0.1 -k temporal_visibility setup-schema -v 0.0
 	./temporal-cassandra-tool --ep 127.0.0.1 -k temporal_visibility update-schema -d ./schema/cassandra/visibility/versioned
 
+install-schema-mysql-pre5720: bins
```

Comment:
do you think we need to test running with pre-5.7.20 mysql in our release pipelines?

Reply:
I think we may need to chat and decide what versions we plan on supporting longer term

Reply:
I am not very familiar with MySQL, but my version is 8.0.19. Why do we need to support 5.7?